ca_bringup/launch/minimal_ros_control.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns" value="create$(arg robot_id)"/>
+
+  <!-- Robot description -->
+  <include file="$(find ca_description)/launch/create_description.launch" ns="$(arg ns)">
+    <arg name="robot_id" value="$(arg robot_id)"/>
+  </include>
+
+  <!-- Load robot's hw interface -->
+  <node name="robot_hw_interface" pkg="create_control" type="create_control_node" ns="$(arg ns)"
```

Comment:
Shouldn't this be moved inside `create_control`? Maybe it will affect the simulation so you can do this:

```
# File: create_control_hardware.launch
--> Run robot_hw_interface
--> Call create_control.launch
```

Reply:
Done